src/stream/ws/ws.c

```diff
@@ -81,6 +81,8 @@ static ssize_t stream_ws_copy_payload(struct stream_ws* ws, void* dst,
 
 static ssize_t stream_ws_process_ping(struct stream_ws* ws, size_t offset)
 {
+	struct rcbuf* rcbuf;
```

Comment:
Nit: I prefer to define variables in-place unless there's a `goto` that prevents it.